listen_test.go

```diff
@@ -145,6 +147,38 @@ func Test_Listen_TLS(t *testing.T) {
 	}))
 }
 
+// go test -run Test_Listen_Acme_TLS
+func Test_Listen_Acme_TLS(t *testing.T) {
+	// Certificate manager
+	m := &autocert.Manager{
+		Prompt: autocert.AcceptTOS,
+		// Replace with your domain
+		HostPolicy: autocert.HostWhitelist("example.com"),
+		// Folder to store the certificates
+		Cache: autocert.DirCache("./certs"),
```

Comment:
Create a random temp dir and use that as your DirCache.

See here: https://pkg.go.dev/os#MkdirTemp

This will allow running the tests in parallel